docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -442,22 +416,86 @@ Using OR value of 1.89 and P as 0.3
+
+(1) RR = 1.9 (95%CI 1.7-2.0) from Pharoah et al. Int. J. Cancer, 1997
+(2) :math:`P_{fh{s}}` = 0.02
+(3) PAF= :math:`\frac{P_{fh{s}}(RR-1)}{1+P_{fh{s}}(RR-1)}`
+(4) 1-PAF= 
+
+
+(5) :math:`i_{DCIS{|fh1}} =  i_{DCIS}\times(1-PAF)\times RR`
+(6) :math:`i_{DCIS{|fh0}} =  i_{DCIS}\times(1-PAF)`
+
+
+:underline:`2. From susceptable S state to LCIS state`
 
-  .. todo::
-    - distribution of family history
-    - relative risk of family history and breast cancer DCIS/LCIS and stage 1+?
+.. image:: S_to_LCIS.svg
 
-  .. note:: 
-    - GBD risk factors will not be modelled
+- Let family history be denoted as fh
+- Let the prevalence of family history **among the S state population** be :math:`P_{fh{s}}`
+- Let i_LCIS be the overall incidence from S state to LCIS state (:ref:`see breast cancer with stage 0 model for i_LCIS value <2017_cancer_model_breast_cancer_with_stage_0>`)
+- Let incidence among those with family history be  :math:`i_{LCIS{|fh1}}`
+- Let incidence among those without family history be :math:`i_{LCIS{|fh0}}`
+- Let PAF be the population attributable fraction of family history on LCIS among the S population
+- Let RR be the ratio of the probability of developing the outcome LCIS in the exposed to family history group versus the probability of developing the outcome LCIS in the unexposed to family history group among the S state population (we are using the RR for breast cancer for this value, see note below).
+
+(1) RR = 1.9 (95%CI 1.7-2.0) from Pharoah et al. Int. J. Cancer, 1997
+(2) :math:`P_{fh{s}}` = 0.02
+(3) PAF= :math:`\frac{P_{fh{s}}(RR-1)}{1+P_{fh{s}}(RR-1)}`
+(4) 1-PAF= 
+
+(5) :math:`i_{LCIS{|fh1}} =  i_{LCIS}\times(1-PAF)\times RR`
+(6) :math:`i_{LCIS{|fh0}} =  i_{LCIS}\times(1-PAF)`
+
+.. note::
+
+  - The value of RR we use in the above is actually for family history (exposure) to breast cancer (outcome). We are using the RR for breast cancer outcome instead of DCIS or LCIS because are unable to obtain a relative risk from DCIS or LCIS state to breast cancer among the DCIS or LCIS population respectively. 
+  - In uSing the RR for breast cancer outcome as the RR for DCIS/LCIS outcome, we over-estimate the DCIS cases detected by screening and under-estimate breast cancer cases detected by screening (this is because those with family history are screened twice as often). This means it will under-estimate the total pay-out value. (assuming there is a relative risk > 1 of family history on breast cancer from DCIS and LCIS state in reality).
+  - This value is also currently a stand-in value which was taken from Pharoah et al. The research team will derive an RR with a meta-analysis. 
+
+.. todo::
+
+  if the bias stated above is unclear, I can write out a numberical example to illustrate this
```

Comment:
It would be helpful to have some idea of how much bias this produces, too.  Can you think of a way to quantify that?

Reply:
yes, I can try to give a real example. I'll do that in another PR

Reply:
What is d .bkp files?